.github/workflows/ci-test-go.yml

```diff
@@ -48,14 +53,30 @@ jobs:
         if: ${{ inputs.rootless-docker }}
         run: sudo rm -rf /var/run/docker.sock
 
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-            go-version: ${{ inputs.go-version }}
+            go-version: '${{ inputs.go-version }}'
+            cache-dependency-path: '${{ inputs.project-directory }}/go.sum'
         id: go
 
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v3
+      - name: golangci-lint
+        # TODO: Remove each example/module once it passes the golangci-lint
+        if: ${{ inputs.platform == 'ubuntu-latest' && inputs.go-version == '1.20.x' && !contains(fromJSON('["examples/cockroachdb", "examples/spanner", "examples/toxiproxy", "modules/compose", "modules/pulsar", "modules/redis"]'), inputs.project-directory) }}
```

Comment:
What is the reason those modules need to be excluded? Are they hard to fix?

Reply:
This PR has already several changes concerning the evolution of the workflow so I preferred not to include the fix of every linters for each module in there. That would be to much for me.

Reply:
Indeed, no questions about that 😊 just wanted to double check with you